rust/lang/security/audit/command-injection-process-builder.rs

```diff
@@ -0,0 +1,5 @@
+//ruleid: dangerous-spawn-process
```

Comment:
Could you also add True Negative tests, so tests indicated with `//ok: <ruleid>`